transport/control.go

```diff
@@ -46,8 +46,8 @@ const (
 	// The default value of flow control window size in HTTP2 spec.
 	defaultWindowSize = 65535
 	// The initial window size for flow control.
-	initialWindowSize             = defaultWindowSize      // for an RPC
-	initialConnWindowSize         = defaultWindowSize * 16 // for a connection
+	initialWindowSize             = defaultWindowSize * 2048      // for an RPC
+	initialConnWindowSize         = defaultWindowSize * 16 * 2048 // for a connection
```

Comment:
A larger window size means more memory that can be buffered by gRPC without being acknowledged by the application. In addition to the extra memory usage, you've likely hurt latency for small requests sharing the same connection. 

This change increases the stream and connection window sizes by 2048x for only a 5.5x throughput improvement. I think there is a less radical change that can provide most of the benefit. In my testing with CockroachDB, bumping the stream and connection window sizes to 2MB (increase of 32x) was the minimum size that provided close to max throughput on a link with 60ms RTTs. As a stop-gap until the window sizes dynamically adjust, I'd be much more comfortable with a change which left the defaults unchanged but made the stream and window sizes configurable.